src/account/Support.tsx

```diff
@@ -19,23 +21,29 @@ const onPressContact = () => {
 
 const Support = () => {
   const { t } = useTranslation()
+  const { externalLinks } = getDynamicConfigParams(DynamicConfigs[StatsigDynamicConfigs.APP_CONFIG])
 
   return (
     <SafeAreaView>
       <CustomHeader left={<BackButton />} title={t('help')} style={styles.paddingHorizontal} />
       <ScrollView>
-        <SettingsItemTextValue
-          testID="FAQLink"
-          title={t('faq')}
-          onPress={openExternalLink(FAQ_LINK)}
-          isExternalLink
-        />
-        <SettingsItemTextValue
-          testID="ForumLink"
-          title={t('forum')}
-          onPress={openExternalLink(FORUM_LINK)}
-          isExternalLink
-        />
+        {!!externalLinks.faq && (
+          <SettingsItemTextValue
+            testID="FAQLink"
+            title={t('faq')}
+            onPress={openExternalLink(externalLinks.faq)}
+            isExternalLink
+          />
+        )}
+        {!!externalLinks.forum && (
```

Comment:
i think it's reasonable to assume that not all apps would want an faq and forum link, so we can conditionally render them based on the availability of the links. in other flows (e.g. swap, cash in) we have the ability to turn off the whole flow at a higher config level